src/components/InputBook.js

```diff
@@ -0,0 +1,55 @@
+import { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import cx from 'classnames';
+import styles from 'styles/InputBook.module.css';
+import { addBook } from 'redux/books/booksSlice';
+import { v4 as uuidv4 } from 'uuid';
+
+const InputBook = () => {
+  const dispatch = useDispatch();
+  const [title, setTitle] = useState('');
+  const [author, setAuthor] = useState('');
+  const itemId = uuidv4();
+  const category = '';
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    if (title.trim() && author.trim()) {
+      dispatch(addBook({
+        itemId, title, author, category,
+      }));
+    }
+    setTitle('');
+    setAuthor('');
+  };
+  return (
+    <>
+      <div className={`mb-2rem bold ${styles.add__book}`}>
+        <h2 className="secondary fs-1-2rem mb-0-5rem">ADD NEW BOOK</h2>
+        <form onSubmit={handleSubmit} className={styles.book__addform}>
+          <input
+            className={cx(styles.addform__input, styles.flex - 1)}
+            type="text"
+            value={title}
+            placeholder="Book Title"
+            onChange={(e) => {
+              setTitle(e.target.value);
+            }}
+            size="90"
+          />
+          <input
+            className={styles.addform__input}
+            type="text"
+            value={author}
+            placeholder="Author"
+            onChange={(e) => {
+              setAuthor(e.target.value);
+            }}
+            size="40"
+          />
+          <button type="submit" className={`blue-button bold ${styles.btn_width}`}>ADD BOOK</button>
+        </form>
+      </div>
```

Comment:
- [ ] While working on the changes the previous code reviewer requested about adjusting the spacing between the book container and the form and also the spacing between elements in the form, Your project is having horizontal overflow and your `Add Book` button is not showing on the page unless the page is scrolled horizontally. Please make sure you remove the horizontal overflow and make the design looks exactly like he Zeplin template:relaxed: See from the video below :point_down: 

[Uploading Screencast from 03-04-2023 02:45:04 AM.webm…]()

Reply:
Hi Desmond, thanks for pointing it out. I made that form responsive now.